projects/vdk-heartbeat/src/vdk/internal/heartbeat/vdk_heartbeat_data_job/trino_ingestion/requirements.txt

```diff
@@ -0,0 +1 @@
+vdk-trino
```

Comment:
I was wondering if we can generalize the job. The only reason to call it trino ingestion is because of this. But ingestion is entirely configurable. And for some other deployments, users of vdk-heartbeat may want to test impala ingestion or HTTP ingestion. So can we make it configurable ? 

In order to make it configurable we need 
* ingest_method_default to be set 
* the correct plugin to be installed

Reply:
I don't think it is in the scope of this issue. It sounds like a future improvement. What do you think?

Reply:
Can you please link the issue? It's a good point I should have asked first about what is the problem we are trying to solve here?

Reply:
Ok. we synced we do not have issue for that. 

But I think we have it currently pretty generalized. 
1. Users can configure ingest_method and db default type in vdk heartbeat config. 
2. Can technically configure plugin as both local and cloud installation distribution vdk is configurable (pip install or through the helm chart for cloud runs) .